ggml/src/ggml-sycl/dpct/helper.hpp

```diff
@@ -870,7 +873,12 @@ namespace dpct
             }
             return -1;
         }
-
+	inline int get_rank() { return _rank; }
```

Comment:
These new functions have no relationship with DPCT.
It's better to move the ggml-sycl/src.
Recommend to reduce the dependence on DPCT code.